realm_dart/README.md

```diff
@@ -0,0 +1,47 @@
+## A simple command-line application using Realm Dart SDK
+
+## Setup
+
+* Dart SDK 2.12 stable is required from https://dart.dev/.
+
+    **Do not use the Dart SDK downloaded with Flutter 2.0 since it has issues and will not be able to run this sample**
```

Comment:
this file should be updated. take hints from realm-dart\README.md

Reply:
Done.